### ISSUE TITLE
When scrolled up in chat indicate if new messages have arrived.

### ISSUE THREAD

**Issue by creesch** [thelounge/lounge#1936]
Basically when you have the lounge open now and you are in a channel while being scrolled up you do not get any indication of new messages having arrived. 

It would be nice to have something like this displayed

![image](https://user-images.githubusercontent.com/2502190/34565136-f2167156-f159-11e7-89e0-dcd1750127bc.png)

Where on clicking you get scrolled to the first new message.

**Comment by astorije** [comment]
Yes to this! I would make it a bit more subtle though, similar to what Telegram does.

When scrolled up, Telegram has the following, and clicking gets you at the bottom. It would be very useful for us as well I think, as channels quickly get full when not paying attention and one has to scroll up a lot, then all the way down.

<img width="188" alt="screen shot 2018-01-04 at 19 50 19" src="https://user-images.githubusercontent.com/113730/34591111-28de2b50-f189-11e7-877b-2d29aac1125d.png">

Then, if a new message arrives when not scrolled to bottom, it adds an unread marker:

<img width="187" alt="screen shot 2018-01-04 at 19 50 09" src="https://user-images.githubusercontent.com/113730/34591124-3930c954-f189-11e7-8b76-d0ae43ea2870.png">
